app/azure/views/create-container-instance.view/create-container-instance.html

```diff
@@ -7,7 +7,7 @@
   <div class="col-sm-12">
     <rd-widget>
       <rd-widget-body>
-        <form class="form-horizontal" autocomplete="off" name="aciForm">
+        <form class="form-horizontal" autocomplete="off" name="$ctrl.aciForm">
```

Comment:
`name` isn't a controller bound variable it's only the form reference name. Revert the changes applied to `aciForm` usage.

Reply:
Anticipating future form validation done in controller instead of in html